[ISSUE 464] Add cluster point process call generation script

jardiamj opened the pull request:
Closes #464

This PR adds a python script under Tools for modeling the call arrival process as a cluster point process.

stiber reviewed (changes requested):
Would be good to add README.md file(s) that describe the files within the directory.

jardiamj commented:
> Would be good to add README.md file(s) that describe the files within the directory.

Reorganized the script directories, renamed them to be more descriptive of what they contain, and added a README.md file.
